sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes \
+ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="2001-2002 gbvalor"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/glucas/Glucas-$portVersion.tar.gz"
+CHECKSUM_SHA256="fde422220245788c0d8aeaec6a89f26e849905b1417c5e6ff1a01f56c595cadc"
+PATCHES="glucas-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	glucas = $portVersion
+	cmd:Glucas = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+BUILD_REQUIRES="
```

Comment:
```diff
 REQUIRES="
 	haiku
 	"
+
 BUILD_REQUIRES="
```
(We like to have an empty line before `BUILD_REQUIRES`.)

([HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))